src/device-registry/package.json

```diff
@@ -54,6 +64,7 @@
         "kafka-node": "^5.0.0",
         "kafkajs": "^1.16.0",
         "log4js": "^6.3.0",
+        "module-alias": "^2.2.2",
```

Comment:
@BenjaminSsempala 
the newly added module to support short path names while importing things. Just to make our code a little neater. :) More: https://www.npmjs.com/package/module-alias